### ISSUE TITLE
Filename has no html extension bei 

### ISSUE THREAD

**Issue by hansii99** [downthemall/downthemall#33]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by nmaier** [comment]
Yeah, we cannot change the file name once the download started, and we cannot query the mime type to determine a good file extension until we start the download. So this WebExtension limitation is really a catch-22

**Comment by nmaier** [comment]
Oh, and just assuming everything without an extension is a html is not good idea either. We do rename to `index.html` if the URL ends in a slash, as a compromise, but that is as far as I would go with this.

**Comment by hansii99** [comment]
Now have done some Tests (FF 70b, DTA 4.0.9).
Seems working now, the html extensions are generated as expected (DTA3).
Thank you
